float32 cell_size
---
uint8[] map